Doc/whatsnew/3.3.rst

```diff
@@ -1504,7 +1504,7 @@ argument taking an iterable of handlers to be added to the root logger.
 A class level attribute :attr:`~logging.handlers.SysLogHandler.append_nul` has
 been added to :class:`~logging.handlers.SysLogHandler` to allow control of the
 appending of the ``NUL`` (``\000``) byte to syslog records, since for some
-deamons it is required while for others it is passed through to the log.
+daemons it is required while for others it is passed through to the log.
```

Comment:
Please fix the same typo in `Lib/test/test_threading.py`.

Reply:
@serhiy-storchaka apologies, are you saying that `daemons` is correct here? 
and in regards to `Lib/test/test_threading.py` i should fix: 
https://github.com/python/cpython/blob/1023dbbcb7f05e76053486ae7ef7f73b4cdc5398/Lib/test/test_threading.py#L430
so that it shows:
`def test_daemon_param(self):`